lms/static/js/student_account/views/AccessView.js

```diff
@@ -190,7 +191,7 @@ var edx = edx || {};
             this.element.scrollTop( $anchor );
 
             // Update url without reloading page
-            History.pushState( null, document.title, '/account/' + type + '/' );
+            History.pushState( null, document.title, '/account/' + type + queryParams );
```

Comment:
It's better to keep the trailing slash, so the user doesn't need to wait for the redirect.